Holovibes/assets/ui/mainwindow.ui

```diff
@@ -5772,38 +5785,6 @@ li.checked::marker { content: &quot;\2612&quot;; }
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>ImportStartIndexSpinBox</sender>
-   <signal>valueChanged()</signal>
-   <receiver>ImportPanel</receiver>
-   <slot>update_input_file_start_index()</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>20</x>
-     <y>20</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>20</x>
-     <y>20</y>
-    </hint>
-   </hints>
-  </connection>
-  <connection>
-   <sender>ImportEndIndexSpinBox</sender>
-   <signal>valueChanged()</signal>
-   <receiver>ImportPanel</receiver>
-   <slot>update_input_file_end_index()</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>20</x>
-     <y>20</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>20</x>
-     <y>20</y>
-    </hint>
-   </hints>
-  </connection>
```

Comment:
The issue with valueChanged() comes with this PR.
When in moments mode, the batch size is force set to 3 so the indexes all need to be multiples of 3. If the valueChanged signal is kept, some numbers are impossible to type with the keyboard because they are not multiples of 3 and are immediately changed to one (ex: 2100, a multiple of 3 which starts with a non-multiple of 3).

Reply:
Ha yes i see what you mean